renderer.py

```diff
@@ -15,23 +15,41 @@ def render(self):
         # outputBox.grid(row=0, column=0)
 
         # Create number buttons
-        button0 = Button(root, text="0", bg="#2D2D2D", fg="#EEEEEE", padx=5, pady=5)
-        button0.config(command=partial(self.calculator.append_expression, button0.cget("text")))
-        button0.grid(row=4, column=1)
+        button_0 = Button(root, text="0", bg="#2D2D2D", fg="#EEEEEE", width=3, pady=5)
+        button_0.config(command=partial(self.calculator.append_expression, button_0.cget("text")))
+        button_0.grid(row=4, column=0)
 
         currentRow = 3
```

Comment:
currentRow and currentColumn so they follow convention and the naming of the other variables in the code.